modules/mobile/app/controllers/mobile/v0/community_care_providers_controller.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    class CommunityCareProvidersController < ApplicationController
+      DEFAULT_PAGE_NUMBER = 1
+      DEFAULT_PAGE_SIZE = 10
+      RADIUS = 60 # default value used in web app
+      SPECIALTIES = {
+        primaryCare: %w[207QA0505X 363LP2300X 363LA2200X 261QP2300X],
+        foodAndNutrition: %w[133V00000X 133VN1201X 133N00000X 133NN1002X],
+        podiatry: %w[213E00000X 213EG0000X 213EP1101X 213ES0131X 213ES0103X],
+        optometry: %w[152W00000X 152WC0802X],
+        audiologyRoutineExam: %w[231H00000X 237600000X 261QH0700X],
+        audiologyHearingAidSupport: %w[231H00000X 237600000X]
+      }.with_indifferent_access.freeze
+
+      def index
+        ccps = ppms_api.facility_service_locator(locator_params)
+        community_care_providers = ccps.map do |record|
+          Mobile::V0::CommunityCareProvider.new(
+            id: record.id,
+            name: record.provider_name,
+            address: Address.new(
+              street: record.address_street,
+              city: record.address_city,
+              state: record.address_state_province,
+              zip_code: record.address_postal_code
+            ),
+            distance: record.miles
+          )
+        end
+        page_records, page_meta_data = paginate(community_care_providers)
+        serialized = Mobile::V0::CommunityCareProvidersSerializer.new(page_records, page_meta_data)
+        render json: serialized, status: :ok
+      end
+
+      private
+
+      def ppms_api
+        FacilitiesApi::V1::PPMS::Client.new
```

Comment:
I know we recently updated some appointments logic to use `VAOS::V2::MobileFacilityService` is that also something we could do here?

Reply:
I'll look into that. Thanks for pointing it out.

Reply:
I don't think I can use that here. This endpoint isn't covered by that service.